packages/browser/src/plugins/segmentio/normalize.ts

```diff
@@ -25,6 +27,16 @@ export function normalize(
     json._metadata = { failedInitializations: failed }
   }
 
+  if (ctx != null) {
+    const retryCount = analytics.queue.queue.getAttempts(ctx)
+    if (retryCount > 1) {
+      json._metadata = {
+        ...json._metadata,
```

Comment:
I didn't think the main event queue retries are related to the retries in the client, so not sure that this would work? Also, this metadata field isn't super 

I would take this our, or write tests for this.

Reply:
pushWithBackoff calls updateAttempts, and I do have a check of getAttempts in the `retries on 500` test, but I think more explicit tests is a good idea.

Reply:
Noticed that this doesn't work for batch.  I'll see if I can do that, but maybe ok without as well.

Reply:
Figured out the queues, have the attempts incrementing independently, and added test for the output.